rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -247,6 +247,7 @@ http {
     geoip2 /etc/nginx/geoip/GeoIP2-ISP.mmdb {
         $geoip2_isp isp;
         $geoip2_isp_org organization;
```

Comment:
should we add `source=$remote_addr` to these too?

Reply:
sure, I will add that.